sda-download/api/api.md

```diff
@@ -0,0 +1,54 @@
+# API
```

Comment:
Thanks for the review :+1: !
Yes... I thought it would be nice to have it under `api/` for consistency with the services in `sda/cmd`. But you are right that it's better to have things in one place, so I merged the `docs/API.md` with the new one in
c49f140abc9f2597733b53b1bc17cdcd424b108c. Please have look!